library/general/block-endpoint-edit-role/template.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: blockendpointeditrole
+  annotations:
+    description: Clusterrole/system:aggregate-to-edit should not allow endpoint permissions
```

Comment:
Can we add a link to the CVE here, so users can assess whether this is still relevant in the future?